resources/js/Pages/Dashboard.vue

```diff
@@ -1,25 +1,237 @@
 <script setup>
 import AuthenticatedLayout from '@/Layouts/AuthenticatedLayout.vue';
 import { Head } from '@inertiajs/vue3';
-</script>
 
+// defineProps({ events: Array })
+defineProps({ credit_points_total: Number,
+                    credit_points_achieved: Number,
+                    modules: Array,
+                    modules_done: Number,
+                    modules_total: Number,
+                    grade_average: Number,
+                    tasks: Array,
+                    events: Array
+})
+</script>
 <template>
-    <Head title="Übersicht" />
-
+<Head title="Übersicht" />
     <AuthenticatedLayout>
-        <template #header>
-            <h2 class="font-semibold text-xl text-gray-800 dark:text-gray-200 leading-tight">Dashboard</h2>
-        </template>
-
-        <div class="py-12">
-            <div class="max-w-7xl mx-auto sm:px-6 lg:px-8 space-y-6">
-                <div class="p-4 sm:p-8 bg-white dark:bg-gray-800 overflow-hidden shadow-sm sm:rounded-lg">
-                    <div class="p-6 text-gray-900 dark:text-gray-100">You're logged in!</div>
-                </div>
-                <div class="p-4 sm:p-8 bg-white dark:bg-gray-800 overflow-hidden shadow-sm sm:rounded-lg">
-                    <div class="p-6 text-gray-900 dark:text-gray-100">Hier siehst du deine Übersicht.</div>
-                </div>
-            </div>
-        </div>
-    </AuthenticatedLayout>
+		<template #header>
+			<h1 class="font-semibold text-xl text-gray-800 leading-tight">Übersicht</h1>
+		</template>
+		<!-- 0 - This is the outermost Div -->
+		<div class="py-6 flex justify-center items-center">
+
+			<!-- 1 - This is the second outermost Div-->
+			<div class="max-w-7xl mx-auto space-y-6 h-screen">
```

Comment:
Remove 'h-screen' for the flex container that sets the background to extend if needed